dashboard/test/ui/features/step_definitions/steps.rb

```diff
@@ -1402,6 +1402,14 @@ def create_user(name, url: '/users.json', code: 201, **user_opts)
   end
 end
 
+And(/^I join the section I already own$/) do
+  page_load(true) do
```

Comment:
Does this differ from "I attempt to join the section" only by this page_load?  If so, why is having this page_load important?